osu.Game/Overlays/Settings/Sections/SizeSlider.cs

```diff
@@ -9,8 +10,9 @@ namespace osu.Game.Overlays.Settings.Sections
     /// <summary>
     /// A slider intended to show a "size" multiplier number, where 1x is 1.0.
     /// </summary>
-    internal class SizeSlider : OsuSliderBar<float>
+    internal class SizeSlider<T> : OsuSliderBar<T>
+        where T : struct, IEquatable<T>, IComparable<T>, IConvertible, IFormattable
     {
-        public override LocalisableString TooltipText => Current.Value.ToString(@"0.##x");
+        public override LocalisableString TooltipText => Current.Value.ToString(@"0.##x", null);
```

Comment:
What is this `null`?

Reply:
A value to use the current culture info, according the [docs](https://docs.microsoft.com/en-us/dotnet/api/system.iformattable.tostring?view=net-6.0#system-iformattable-tostring(system-string-system-iformatprovider)):

![CleanShot 2022-02-16 at 10 24 53](https://user-images.githubusercontent.com/22781491/154216064-8f86fdc7-b63f-41cc-868f-d6085e89ce47.png)

Can replace with `NumberFormatInfo.CurrentInfo` if that reads better.

Reply:
Please do. null looks very very weird.